docs/apis/management/user_management.yaml

```diff
@@ -224,6 +224,88 @@ paths:
                   message:
                     type: string
                     description: Error message for an unprocessable entity
+  /roles:
+    get:
+      summary: Get roles
+      description: Retrieve roles data for account with list of users for each role
+      tags:
+        - Managing roles
+      responses:
+        "200":
+          description: Successful response containing the list of roles with capabilities
+            and users for each role
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  roles:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        name:
+                          type: string
+                          description: The display name of the role
+                        type:
+                          type: string
+                          enum:
+                            - predefined
+                            - custom
+                          description: Describes whether the role is custom or pre-defined
+                        capabilities:
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              section:
+                                type: string
+                                description: The name of a section of capabilities
+                              capabilities:
+                                type: array
+                                items:
+                                  type: object
+                                  properties:
+                                    name:
+                                      type: string
+                                      description: The name of the capability
+                                    access_level:
+                                      type: string
+                                      enum:
+                                        - Managed
+                                        - Read
+                                        - No access
+                        users:
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              email:
```

Comment:
Do you want to add regex validations of an email?

Reply:
We don't have it anywhere else, so I guess not (for now)